### PR TITLE
Improved (podman - remote-containers) documentation

### PR DESCRIPTION
**Pull request by ckupe**
Signed-off-by: Chris Kuperstein <ckuperst@redhat.com>

Note: These edits were written with [Mark Text on Linux](https://marktext.app/) and may contain UNIX line endings. Changes deviating from target branch in this PR are generally confined to the section "Can I use Podman instead of Docker?" in `docs/remote/containers.md`

List of changes:

- Added paragraph<sup>[1](https://github.com/microsoft/vscode-docs/compare/master...ckupe:podman-symlink-note#diff-b80970227c86e646206f3f8dadb7db23R1172)</sup> describing and linking relevant issue comments from maintainers of the remote-container extension regarding the need to softlink docker to podman in order for the extension to not request docker to be installed, or to throw daemon status errors.
- Added paragraph<sup>[2](https://github.com/microsoft/vscode-docs/compare/master...ckupe:podman-symlink-note#diff-b80970227c86e646206f3f8dadb7db23R1178)</sup> providing new supporting material regarding selinux configurations when using podman instead of docker as the path executable for the remote-container extension, to include a link to policies and labels for consumers to use in order for projects to successfully mount to built containers
- Minor style edit to the first referenced command palette function<sup>[3](https://github.com/microsoft/vscode-docs/compare/master...ckupe:podman-symlink-note#diff-b80970227c86e646206f3f8dadb7db23R1180)</sup> in this subsection to be [more in line with the documentation contribution guide](https://github.com/microsoft/vscode-docs/blob/master/CONTRIBUTING.md#text-formatting)

### CODE REVIEW

File: docs/remote/containers.md [hunk 1]
```diff
@@ -1163,11 +1165,19 @@ See [Docker Desktop for Windows tips](/docs/remote/troubleshooting.md#docker-des
...
+Podman 1.9+ is mostly compatible with Docker's CLI commands and therefore generally does work if you update the setting **Remote > Containers: Docker Path** to `podman` on Linux. 
 
 ![Docker Path setting](images/containers/docker-path-setting.png)
 
-However, certain tricks like [Docker-from-Docker do not work](https://github.com/containers/libpod/issues/4056#issuecomment-535511841) due to limitations in Podman. This affects the **Remote-Containers: Try a Sample...** and **[Remote- Containers: Open repository in container...](#quick-start-open-a-git-repository-or-github-pr-in-an-isolated-container-volume)** commands.
+You [may need to symbolically link](https://github.com/microsoft/vscode-remote-release/issues/116#issuecomment-619329668[Podman support · Issue #116 · microsoft/vscode-remote-release · GitHub](https://github.com/microsoft/vscode-remote-release/issues/116#issuecomment-619329668)) docker to podman in order for the extension to function when docker and docker-compose are not installed on the system. If not linked, the extension will require docker to be installed even if you change the executable docker path settings of the extension mentioned above.
```
Comment (ckupe) [inline docs/remote/containers.md:1172]: Begin PR context here, ends at [line 1181](https://github.com/microsoft/vscode-docs/pull/3847/files#diff-b80970227c86e646206f3f8dadb7db23R1181).

Comment (chrmarti) [inline docs/remote/containers.md:1172]: With the above setting the symbolic link is no longer needed.

File: docs/remote/containers.md [hunk 2]
```diff
@@ -1163,11 +1165,19 @@ See [Docker Desktop for Windows tips](/docs/remote/troubleshooting.md#docker-des
...
+
+```
+sudo ln -s /usr/bin/docker /usr/bin/podman
+```
+
+Depending on the linux distribution you install podman on, you may also need to configure selinux for the extension to be able to mount project folders. For Fedora 31+ or container centric linux distributions such as CoreOS and Fedora CoreOS, there may already be a `container-selinux` package present that ships with the correct selinux configs for rootless operation of podman. For other distributions not based on EL that still use selinux, [refer to container-selinux](https://github.com/containers/container-selinux) for appropriate selinux labels and policies that can be applied to your project directories in order to mount projects while selinux is set to enforcing mode.
```
Comment (chrmarti) [inline docs/remote/containers.md:1178]: Casing: Linux, SELinux, Podman.

What is EL?

File: docs/remote/containers.md (hunk 1 above)
Comment (ckupe) [reply docs/remote/containers.md:1172]: @chrmarti yesterday I went through and installed the supported .rpm based installation of VSCode and the latest extension version and I still needed to create a soft link in order for it not to ask for docker to be installed. I'll add additional notes in the appropriate linked issue to provide more insight on that case, which is why I included this section with a link to that issue. 

This issue may also affect the flathub/flatpak repackages of VSCode for a while-- so even if its patched with the latest version of the extension and IDE, the behavior may still persist for some time until the unsupported community distributions catch up.

File: docs/remote/containers.md (hunk 2 above)
Comment (ckupe) [reply docs/remote/containers.md]: Requested changes made

**Comment by ckupe** [comment]
@chrmarti I rebased this PR branch off master and locally configured `git config --global core.autocrlf false` since the documentation repository `.gitattributes` file contains `* text=auto eol=lf` as a cross-platform normalization setting. I re-added and committed with `git add . --renormalize`, so not sure what else could cause the excess whitespacing.

**Comment by gregvanl** [comment]
Hi @ckupe Unfortunately this PR has gotten too far away from the current state of vscode-docs. Would you be able to prepare a new PR with just your proposed changes? Thanks

**Comment by PavelSosin-320** [comment]
@gregvanl Please, document:
1.It is possible to run Podman using Ubuntu 20.04 WSL distro. Kube project is merged with Ubuntu 20.04 and provides fully functional Podman
To run Podman in "remote" mode 3 steps to do:
Install Podman CLI for Windows
On the Ubuntu side do
sudo podman system service -t 0 tcp:localhost:3375, or any other available port
On the Windows side use 
podman --remote --url tcp://localhost:3375 and use modern Podman in the remote mode instead of obsolete Docker!

-

**Pull request closed without merging**